docs/tutorial_advanced.rst

```diff
@@ -483,7 +516,7 @@ Let's see how we can rewrite the :class:`MemoryLimitTest` using the ``memory`` r
    :lines: 26-38
    :emphasize-lines: 11-13
 
-The extra resources that the test needs to obtain through its scheduler are specified in the :attr:`extra_resources <reframe.core.pipeline.RegressionTest.extra_resources>` attribute, which is a dictionary with the resource names as its keys and another dictionary assigning values to the resource placeholders as its values.
+The extra resources that the test needs to obtain through its scheduler are specified in the :attr:`~reframe.core.pipeline.RegressionTest.extra_resources` attribute, which is a dictionary with the resource names as its keys and another dictionary assigning values to the resource placeholders as its values.
```

Comment:
I don't know where to put, so I put here a fix for line 463.  It should be changed to `This object has an :attr:`options` attribute`